python/send.py

```diff
@@ -323,7 +332,7 @@ def getJobInfo(argv):
             pt = args.pt
             print "LHE: ", LHE
             print "jet pt: ", pt
-            job_dir += "etaTo0.5/" + str(pt) + "GeV/"
+            job_dir += "etaTo"+str(args.etaMax)+"/" + str(pt) + "GeV/"
```

Comment:
this name doesn't correspond to the hard-coded values in lines 517-529.

Reply:
true, i think Clement wanted to replace the hardcoded values. i just added this for the reco to be able to run on earlier produced simu files. but i think we can use the --etaMax to set this also for the physics simulations.
What do you say, @clementhelsens ?